### PR TITLE
Documentation for Mercedes me component

### PR DESCRIPTION
**Pull request by ReneNulschDE**
Documentation for new Mercedes me component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11743

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/binary_sensor.mercedesme.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to integrate Mercedes me binary sensors into Home Assistant."
+date: 2018-01-06 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Binary Sensor
+ha_iot_class: "Cloud polling"
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/binary_sensor.mercedesme.markdown]: `0.61` is already shipped. Please use `0.62`.

File: source/_components/device_tracker.mercedesme.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: "Mercedes me"
+description: "Instructions on for how to integrate Mercedes me into Home Assistant."
+date: 2018-01-06 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Presence Detection
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/device_tracker.mercedesme.markdown]: `0.61` is already shipped. Please use `0.62`.

File: source/_components/mercedesme.markdown [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+title: "Mercedes me"
+description: "Instructions on how to integrate Mercedes car with Mercedes me into Home Assistant."
+date: 2018-01-06 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Hub
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: `0.61` is already shipped. Please use `0.62`.

File: source/_components/sensor.mercedesme.markdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions on how to integrate Mercedes me sensors into Home Assistant."
+date: 2018-01-06 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Sensor
+ha_iot_class: "Cloud Push"
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/sensor.mercedesme.markdown]: `0.61` is already shipped. Please use `0.62`.

File: source/_components/mercedesme.markdown [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+To use Mercedes me in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+mercedesme:
+  username: email
+  password: password
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/mercedesme.markdown [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+sharing: true
+footer: true
+logo: mercedesme.png
+ha_category: Hub
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+
+The `Mercedes me` component offers integration with the [Mercedes me](https://www.mercedes-benz.com/de/mercedes-me/) cloud service and provides presence detection as well as sensors such as doors, tires, windows and service interval.
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: Missing comma: `windows and service interval` -> `windows, and service interval`

File: source/_components/mercedesme.markdown [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+This component provides the following platforms:
+ - Binary Sensors - such such as windows, tires, doors, lock
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: - `such such` -> `such`

File: source/_components/mercedesme.markdown [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+ - Sensors - such as fuelStatus, service interval, remaining km...
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: - `fuelStatus` -> `fuel status`

File: source/_components/mercedesme.markdown [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+mercedesme:
+  username: email
+  password: password
+```
+
+Configuration variables:
+
+- **username** (*Required*): The email address associated with your Mercedes me account.
+- **password** (*Required*): The password for your given Mercedes me account.
+- **scan_interval** (*Optional*): API polling interval. Minimal value can't be less then 300. (Defaults 300)
```
Comment (frenck) [inline source/_components/mercedesme.markdown]: - `Minimal value can't be less then 300.` -> `The minimal value can't be less than 300.`

**Comment by frenck** [comment]
@ReneNulschDE I've reviewed your documentation. Could you please take a look at my comments?

**Comment by ReneNulschDE** [comment]
@frenck Thank you for the review. I've committed the requested changes.

File: source/_cookbook/configuration_yaml_by_aneisch.markdown [hunk 10]
```diff
@@ -8,6 +8,6 @@ comments: false
 sharing: true
```
Comment (frenck) [inline source/_cookbook/configuration_yaml_by_aneisch.markdown]: Please remove this file. It has nothing to do with this PR.

Create a separate PR for this.

**Comment by frenck** [comment]
@ReneNulschDE Added another RFC.

**Comment by ReneNulschDE** [comment]
@frenck could you give ma a hint how I can delete this file from the PR. I can delete the PR and create new one otherwise.

**Comment by ReneNulschDE** [comment]
@frenck I would like to close this PR and reopen a new one with the same files but on a feature branch. I can't update other docs otherwise. Sorry my mistake... I'll write down 50 times: _"Check that I'm on the right branch!"_

**Comment by frenck** [comment]
@ReneNulschDE Don't worry about that. If that still is an issue when the merge needs to happen, we can always decide to do something else.

**Pull request closed without merging**